pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -894,6 +896,14 @@ public void setBrokerClientAuthenticationParameters(String brokerClientAuthentic
         this.brokerClientAuthenticationParameters = brokerClientAuthenticationParameters;
     }
 
+    public String getBrokerClientTrustCertsFilePath() {
```

Comment:
Should this default to the server trust store if empty?

Reply:
I feel it will make setting the trust store a bit complicated. Right now it is simple to set one trust store for client side and one for server.
But if you feel strongly about it - let me know I will set the default to server trust store.

Reply:
@merlimat - waiting for your reply before merging

Reply:
Sure, sounds good. We can revisit later if needed.